docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -55,19 +55,31 @@ Even in the case that requires some infrastructure investment, `dagster-airflow`
 
 ---
 
-## Limitations
+## Deciding on persistent vs ephemeral airflow database
```

Comment:
Should this be considered a prerequisite (something the user needs to do before they start) or a step in the tutorial itself? Right now this section is sorta hanging out between the Prereqs and the first step, which is a bit odd.

It looks like it should be a step, based on the fact that the user is writing some code and interacting with the `dagster-airflow` library. If that's the case, we'll need to shuffle the order of steps around a bit.

Reply:
👍 great callout.

I moved this into a step right before going to production.